extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerRestClient.java

```diff
@@ -129,7 +132,15 @@ else if (url.startsWith("https://")) {
 					client.register(HttpAuthenticationFeature.basic(
 							getConfig().getUser(),
 							getConfig().getPass()));
-					webTarget = client.target(url);
+					try {
```

Comment:
this getWebTarget() is an issue (we should technically cache the client based on URL, user, and password, or invalidate it when one of those things changes)